atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -69,15 +69,18 @@ def get_koji_upload_dir(workflow):
     return workspace[WORKSPACE_KEY_UPLOAD_DIR]
 
 
-def override_build_kwarg(workflow, k, v):
+def override_build_kwarg(workflow, k, v, platform=None):
     """
     Override a build-kwarg for all worker builds
     """
     key = OrchestrateBuildPlugin.key
+    if not platform:
+        platform = 'None'
 
     workspace = workflow.plugin_workspace.setdefault(key, {})
     override_kwargs = workspace.setdefault(WORKSPACE_KEY_OVERRIDE_KWARGS, {})
-    override_kwargs[k] = v
+    override_kwargs[platform] = override_kwargs.setdefault(platform, {})
```

Comment:
Don't think we need to assign here. Just the `setdefault()` call is enough to add the key.

Reply:
Fixed.